bundle/manifests/pipelines.openshift.io_gitopsservices.yaml

```diff
@@ -34,13 +34,19 @@ spec:
           spec:
             description: GitopsServiceSpec defines the desired state of GitopsService
             properties:
```

Comment:
Not sure why this NodeSelector code is part of this PR?

Reply:
Thanks for the review. I got some code wrong when I rebased my branch. It should be fixed now.